desktop/components/split_test/running_tests.coffee

```diff
@@ -30,7 +30,7 @@ module.exports = {
   onboarding_test:
     key: 'onboarding_test'
     outcomes:
-      control: 50
-      experiment: 50
+      control: 100
+      experiment: 0
```

Comment:
Accidentally rebased over the commit right before this PR. See master https://github.com/artsy/force/blob/master/desktop/components/split_test/running_tests.coffee#L33

Reply:
Ahh `git reflog` + `git reset HEAD@{3}` to the rescue